tests/ui_tests/test_configs.py

```diff
@@ -359,39 +383,66 @@ def test_configs_fields(sdk_client_fs: ADCMClient, config_dict, app_fs):
     6. Check field configuration (depends on expected result dict and bundle configuration"""
 
     config, expected, path = _prepare_config(config_dict)
-    allure.attach.file(
-        "/".join([path, 'config.yaml']),
-        attachment_type=allure.attachment_type.YAML,
-        name='config.yaml',
-    )
+    allure.attach.file("/".join([path, 'config.yaml']), attachment_type=allure.attachment_type.YAML, name='config.yaml')
     field_type = config['config'][0]['type']
 
-    cluster, ui_config = prepare_cluster_and_get_config(sdk_client_fs, path, app_fs)
-
-    fields = ui_config.get_app_fields()
+    cluster, config_page = prepare_cluster_and_config(sdk_client_fs, path, app_fs)
     with allure.step('Check save button status'):
-        save_err_mess = f"Correct status for save button {[expected['save']]}"
-        button_state = ui_config.save_button_status()
-        assert expected['save'] == button_state, save_err_mess
-        if expected['save']:
-            ui_config.save_configuration()
-            ui_config.assert_no_popups_displayed()
+        expected_state = expected['save']
+        assert (
+            not (config_page.config.is_save_btn_disabled()) == expected_state
+        ), f'Save button should{" not " if expected_state is False else " "}be disabled'
+        if expected_state:
+            config_page.config.save_config()
```

Comment:
Will `save_config` raise "red" exception if it fails?

Reply:
how do we know that in UI? It fails if save button is disabled and loading_text isn't hide for 2 sec

Reply:
We can enforce it by wrapping `save_config` in `catch_failed`.
Actually, I'm not sure that it should be done now, because it's a specific part of UI tests that we don't make "element not found"-like cases red, but keep it yellow.
I suggest we've think of this some time else